L/LibPQ/build_tarballs.jl

```diff
@@ -3,58 +3,73 @@
-tzcode_version = "2021e"
+version = v"16.0"
+tzcode_version = "2023c"
 
 # Collection of sources required to build LibPQ
 sources = [
-    ArchiveSource(
-        "https://ftp.postgresql.org/pub/source/v$pg_version/postgresql-$pg_version.tar.gz",
-        "18eff30438703dd7a5f2a7ca916741bce3c99eadb4025bc6268af268e8e909c5"
+    GitSource(
+        "https://github.com/postgres/postgres.git",
+        "c372fbbd8e911f2412b80a8c39d7079366565d67",
     ),
     ArchiveSource(
         "https://data.iana.org/time-zones/releases/tzcode$tzcode_version.tar.gz",
-        "584666393a5424d13d27ec01183da17703273664742e049d4f62f62dab631775",
+        "46d17f2bb19ad73290f03a203006152e0fa0d7b11e5b71467c4a823811b214e7",
+        unpack_target="zic-build",
     ),
 ]
 
 # Bash recipe for building across all platforms
 # NOTE: readline and zlib are not used by libpq
 script = raw"""
-cd $WORKSPACE/srcdir
+cd zic-build
 make CC=$BUILD_CC VERSION_DEPS= zic
+mv zic ../ && cd ../ && rm -rf zic-build
 export ZIC=$WORKSPACE/srcdir/zic
-cd $WORKSPACE/srcdir/postgresql-*/
-if [[ "${target}" == i686-linux-musl ]]; then
-    # Small hack: swear that we're cross-compiling.  Our `i686-linux-musl` is
-    # bugged and it can run only a few programs, with the result that the
-    # configure test to check whether we're cross-compiling returns that we're
-    # doing a native build, but then it fails to run a bunch of programs during
-    # other tests.
-    sed -i 's/cross_compiling=no/cross_compiling=yes/' configure
-fi
-FLAGS=()
-if [[ "${target}" == *-linux-* ]] || [[ "${target}" == *-freebsd* ]]; then
-    FLAGS+=(--with-gssapi)
-    if [[ "${target}" == *-freebsd* ]]; then
-        # Only for FreeBSD we need to hint that we need to libcom_err to get
-        # functions `add_error_table` and `remove_error_table`
-        export LIBS=-lcom_err
-    fi
+export PATH=$WORKSPACE/srcdir:$PATH
+export CFLAGS="-std=c99"
+
+cd postgres
+
+if [[ ${target} == *-apple-* ]]; then
+    ./configure --prefix=${prefix} \
+        --build=${MACHTYPE} \
+        --host=${target} \
+        --with-includes=${includedir} \
+        --with-libraries=${libdir} \
+        --without-readline \
+        --without-zlib \
+        --with-ssl=openssl \
+        "${FLAGS[@]}"
+
+    make -C src/interfaces/libpq -j${nproc}
+    make -C src/interfaces/libpq install
+    make -C src/include install
+
+else
+    meson setup meson_build --prefix=$prefix \
+        --cross-file="${MESON_TARGET_TOOLCHAIN}" \
+        --bindir=${bindir} \
+        --libdir=${libdir} \
+        --includedir=${includedir} \
+        -Dssl=openssl \
+        -Dzlib=disabled \
+        -Dreadline=disabled \
+        -Dtap_tests=disabled \
+        -Dplpython=disabled \
+        -Dplperl=disabled \
+        -Dnls=disabled
+
+    cd meson_build
+    ninja -j${nproc}
+    ninja install
+    cd ../
+
+    if [[ ${target} == *-w64-mingw32 ]]; then
+        mv -v meson_build/src/interfaces/libpq/* ${prefix}/lib
```

Comment:
What are these files?  Shared libraries or what?  Should they be installed in `${prefix}/lib` or `${libdir}`?

Reply:
Uhm, looking [at the log](https://buildkite.com/julialang/yggdrasil/builds/5196#018aa00b-c8c8-4852-9ceb-bedc9e1a4e6a/646-5732) they look import libraries and other dev files, probably ok to keep in `${prefix}/lib`